airbrakes/data_handling/apogee_predictor.py

```diff
@@ -83,7 +83,7 @@ def stop(self) -> None:
         Stops the logging process. It will finish logging the current message and then stop.
         """
         # Waits for the process to finish before stopping it
-        self._prediction_queue.put(STOP_SIGNAL)  # Put the stop signal in the queue
+        self._prediction_queue.put(LoggerSettings.STOP_SIGNAL.value)  # Put the stop signal in the queue
```

Comment:
You do not need an enum for a single value, just use a constant

Reply:
like if we had 3 or 4 different signals we used, then I could see an argument for it